jpa/WFLY-19554_cdi_persistence_ee11.adoc

```diff
@@ -0,0 +1,176 @@
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://jakarta.ee/xml/ns/persistence
+          https://jakarta.ee/xml/ns/persistence/persistence_3_2.xsd"
+        version="3.2">
+
+    <persistence-unit name="my-unit">
+      <provider>org.jpa.SomePersistenceProvider</provider>
+      <jta-data-source>java:server/datasources/ExampleDS</jta-data-source>
+      <class>org.acme.Employee</class>
+      <properties>
+        <property name="jpa.ddl.auto" value="create-drop"/>
+        <property name="jpa.show_sql" value="true"/>
+        <property name="jpa.format_sql" value="true"/>
+      </properties>
+      <!-- CDI integration -->
+      <cdi:scope>com.example.jpa.ACustomScope</cdi:scope>
+      <cdi:qualifier>com.example.jpa.CustomQualifier</cdi:qualifier>
+  </persistence-unit>
+
+</persistence>
+----
+
+=== Additional EntityManagerFactory Properties
+
+The following additional properties correspond to the new elements and properties in the persistence.xml file when using the persistence-3.2 xsd schema and associated persistenceUnitExtType. When any of these properties are specified in the Map parameter passed to the createEntityManagerFactory method, their values override the values of the corresponding elements and properties in the persistence.xml file for the named persistence unit. They also override any defaults that the persistence provider might have applied.
+
+[cols="1,1,1"]
+|===
+|Property|Type|Corresponding element in persistence.xml|
+|jakarta.persistence.qualifiers, String[], qualifier
+|jakarta.persistence.scope, String, scope
+|=== 
+
+
+=== CDI Specification details for background understanding:
+
+Open the [https://jakarta.ee/specifications/webprofile/11/jakarta-webprofile-spec-11.0-m5#cdi-specification-references] for CDI Core Specification links
+
+=== User Stories
+
+Users looking to either update existing applications or create new applications that use CDI Injection to access EntityManagerFactory/EntityManager/CriteriaBuilder/PersistenceUnitUtil/Cache/SchemaManager/Metamodel bean types within their application code.
+This relies on CDI to locate the appropriate bean type instead of the Persistence container Persistence Unit Scope that can also be used referencing a persistence unit.  
+See [https://jakarta.ee/specifications/persistence/3.2/jakarta-persistence-spec-3.2#a12459] for details about the Persistence container Persistence Unit Scope feature.
+
+=== Compatibility between Jakarta Persistence & CDI integration
+
+==== Sharing of persistence contexts via JTA transactions between CDI injected persistence and other ways (e.g. @PersistenceContext)
+
+* What is shared between Container-managed Transaction-scoped Persistence Context and CDI Persistence inject (in either order) on same thread/unitName.  See [https://jakartaee/specifications/persistence/3.2/jakarta-persistence-spec-3.2#a11805]  Question is whether persistence context instances are shared?
```

Comment:
The jakarta.ee link is broken; missing . before ee

Reply:
Thanks Brian, I missed that for sure!